app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -107,14 +113,62 @@
       <% end %>
     <% end %>
 
-    <div id="wrapper" class="<%= "govuk-width-container" unless full_width %>">
-      <%= yield :before_content %>
-      <main class="govuk-main-wrapper" id="content">
-        <%= yield %>
-      </main>
-    </div>
+    <% if show_account_layout %>
+      <div class="<%= "govuk-width-container" unless full_width %>">
+        <% message = capture do %>
+          <%= t("components.layout_for_public.account_layout.feedback.banners.phase_intro") %>
+          <a class="govuk-link" href=<%= "#{Plek.find('account-manager')}/feedback" %>><%= t("components.layout_for_public.account_layout.feedback.banners.phase_link") %></a>
+          <%= t("components.layout_for_public.account_layout.feedback.banners.phase_outro") %>
+        <% end %>
+
+        <%= render "govuk_publishing_components/components/phase_banner", {
+          phase: "alpha",
+          message: message
+        } unless omit_account_phase_banner %>
+
+
+        <div class="govuk-grid-row govuk-main-wrapper">
+          <%= tag.div(class: 'govuk-grid-column-one-third') do %>
+            <%= render "govuk_publishing_components/components/layout_for_public/account-navigation", { page_is: account_nav_location} %>
+          <% end unless omit_account_navigation %>
+
+          <div class="govuk-grid-column-two-thirds">
+            <div id="wrapper">
+              <%= yield :before_content %>
+              <main role="main" id="content">
+                <%= yield %>
+              </main>
+            </div>
+          </div>
+        </div>
+      </div>
+      <% else %>
+        <div id="wrapper" class="<%= "govuk-width-container" unless full_width %>">
+          <%= yield :before_content %>
+          <main class="govuk-main-wrapper" id="content">
+            <%= yield %>
+          </main>
+        </div>
+      <% end %>
 
-    <% unless omit_feedback_form %>
+    <% if show_account_layout %>
+      <%= tag.div(class: "govuk-width-container") do %>
+        <%= tag.div(class: "gem-c-layout-for-public-account-feedback-footer") do %>
+          <%= render "govuk_publishing_components/components/heading", {
+            text: t("components.layout_for_public.account_layout.feedback.banners.title"),
+            heading_level: 2,
+            font_size: "m",
+            margin_bottom: 4,
+          } %>
+
+          <p class="govuk-body govuk-!-margin-bottom-0">
+            <%= t("components.layout_for_public.account_layout.feedback.banners.footer_intro") %>
+            <a href="<%= "#{Plek.find('account-manager')}/feedback" %>" class="govuk-link"><%= t("components.layout_for_public.account_layout.feedback.banners.footer_link") %></a>
+            <%= t("components.layout_for_public.account_layout.feedback.banners.footer_outro") %>
+          </p>
+          <% end %>
+      <% end unless omit_account_feedback_footer %>
```

Comment:
Minor quibble - I think that this `unless` should be at the start of this `do` block so that both of the conditionals are together. For example:
```ruby
<% if show_account_layout >%
  <% unless omit_account_feedback_footer %>
```

Reply:
Updated but I didn't quite do as suggested. I went with this instead:
```
<% if show_account_layout && !omit_account_feedback_footer %>
  <%= render "govuk_publishing_components/components/layout_for_public/account-feedback-footer" %>
<% elsif !omit_feedback_form %>
  <div class="govuk-width-container">
      <%= render "govuk_publishing_components/components/feedback" %>
  </div>
<% end %>
```

Reply:
That's more concise and combines the conditionals - nice.